src/components/Api.js

```diff
@@ -0,0 +1,181 @@
+export default class Api {
+  constructor({ baseURL, authToken }) {
+    this._baseURL = baseURL;
+    this._authToken = authToken;
+  }
+
+  // GET https://around.nomoreparties.co/
+  getCardList() {
+    return fetch("https://around.nomoreparties.co/v1/group-12/cards", {
+      headers: {
+        authorization: this._authToken,
+      },
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      // if the server returns an error, reject the promise
+      return Promise.reject(`Error: ${res.status}`).catch((err) => {
+        console.error(err); // log the error to the console
+      });
+    });
+  }
+
+  getUserInfo() {
+    return fetch("https://around.nomoreparties.co/v1/group-12/users/me", {
+    method: "GET",  
+    headers: {
```

Comment:
You should store the whole headers object in the constructor and reuse it in all the other methods

```js
headers: this._headers
```

Reply:
Done